src/SamplesApp/SamplesApp.UITests/Windows_UI_Xaml_Controls/PopupTests/UnoSamples_Popup_LightDismiss_Tests.cs

```diff
@@ -0,0 +1,195 @@
+			Assert.AreEqual("1", thirdPopupTextBlock.GetDependencyPropertyValue("Opacity")?.ToString());
+			Assert.AreEqual("1", fourthPopupTextBlock.GetDependencyPropertyValue("Opacity")?.ToString());
+
+			// Click on popup button and verify opacity
+			btn4.Tap();
+			Assert.AreEqual("1", firstPopupTextBlock.GetDependencyPropertyValue("Opacity")?.ToString());
+			Assert.AreEqual("1", secondPopupTextBlock.GetDependencyPropertyValue("Opacity")?.ToString());
+			Assert.AreEqual("1", thirdPopupTextBlock.GetDependencyPropertyValue("Opacity")?.ToString());
+			Assert.AreEqual(null, fourthPopupTextBlock.GetDependencyPropertyValue("Opacity")?.ToString());
+
+			// Click on popup button and verify opacity
+			btn4.Tap();
+			Assert.AreEqual("1", firstPopupTextBlock.GetDependencyPropertyValue("Opacity")?.ToString());
+			Assert.AreEqual("1", secondPopupTextBlock.GetDependencyPropertyValue("Opacity")?.ToString());
+			Assert.AreEqual(null, thirdPopupTextBlock.GetDependencyPropertyValue("Opacity")?.ToString());
+			Assert.AreEqual(null, fourthPopupTextBlock.GetDependencyPropertyValue("Opacity")?.ToString());
+
+			// Click on popup button and verify opacity
+			btn4.Tap();
+			Assert.AreEqual("1", firstPopupTextBlock.GetDependencyPropertyValue("Opacity")?.ToString());
+			Assert.AreEqual(null, secondPopupTextBlock.GetDependencyPropertyValue("Opacity")?.ToString());
+			Assert.AreEqual(null, thirdPopupTextBlock.GetDependencyPropertyValue("Opacity")?.ToString());
+			Assert.AreEqual(null, fourthPopupTextBlock.GetDependencyPropertyValue("Opacity")?.ToString());
+
+			// Click on popup button and verify opacity
+			btn4.Tap();
+			Assert.AreEqual(null, firstPopupTextBlock.GetDependencyPropertyValue("Opacity")?.ToString());
+			Assert.AreEqual(null, secondPopupTextBlock.GetDependencyPropertyValue("Opacity")?.ToString());
+			Assert.AreEqual(null, thirdPopupTextBlock.GetDependencyPropertyValue("Opacity")?.ToString());
+			Assert.AreEqual(null, fourthPopupTextBlock.GetDependencyPropertyValue("Opacity")?.ToString());
+		}
+
+		[Test]
+		[AutoRetry()]
+		public void DismissablaPopups_ReverseOrder_Validation()
+		{
+			Run("UITests.Shared.Windows_UI_Xaml_Controls.Popup.Popup_LightDismiss");
+
+			var btn4R = _app.Marked("btn4R");
+
+			var firstPopupTextBlock = _app.Marked("FirstPopupTextBlock");
+			var secondPopupTextBlock = _app.Marked("SecondPopupTextBlock");
+			var thirdPopupTextBlock = _app.Marked("ThirdPopupTextBlock");
+			var fourthPopupTextBlock = _app.Marked("FourthPopupTextBlock");
+
+			// Verify opacity of all popups initially
+			Assert.AreEqual(null, firstPopupTextBlock.GetDependencyPropertyValue("Opacity")?.ToString());
+			Assert.AreEqual(null, secondPopupTextBlock.GetDependencyPropertyValue("Opacity")?.ToString());
+			Assert.AreEqual(null, thirdPopupTextBlock.GetDependencyPropertyValue("Opacity")?.ToString());
+			Assert.AreEqual(null, fourthPopupTextBlock.GetDependencyPropertyValue("Opacity")?.ToString());
```

Comment:
Since you are making those tests many times, you should create a helper method for that. It will make the test easier to read.

Reply:
Yes @manpreetkaurnventive you can create a local method to re-use the code and avoid to copy-paste those lines every time ;)